modules/fscloud/version.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.3.0, <1.6.0"
+
+  required_providers {
+    time = {
```

Comment:
`ibmcloud` provider is missing.    
    ibm = {
      source  = "IBM-Cloud/ibm"
      version = ">= 1.49.0"
    }

Reply:
Added